test/of.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var M = require('..');
```

Comment:
I agree, how would we do that though? How would a use create a `Just` or a `Nothing`?